web/views/recurring_jobs.erb

```diff
@@ -24,8 +25,13 @@
             <a href="<%= root_path %>queues/<%= job_spec.fetch('queue', 'default') %>"><%= job_spec.fetch('queue', 'default') %></a>
           </td>
           <td><%= job_spec['args'] %></td>
+          <td class="text-center">
+            <a class="btn btn-warn btn-xs" href="<%= root_path %>recurring-jobs/<%= name %>/enqueue">
```

Comment:
urls in href are automatically escaped so even though url includes space it's escaped to `%20` etc